tests/test_wave_01.py

```diff
@@ -143,12 +143,13 @@ def test_moves_movie_from_watchlist_to_watched():
     assert len(updated_data["watchlist"]) == 1
     assert len(updated_data["watched"]) == 2
     
-    raise Exception("Test needs to be completed.")
+    #raise Exception("Test needs to be completed.")
     # *******************************************************************************************
     # ****** Add assertions here to test that the correct movie was added to "watched" **********
     # *******************************************************************************************
-
-@pytest.mark.skip()
+    assert movie_to_watch in updated_data["watched"]
+    assert movie_to_watch["title"] not in updated_data["watchlist"]
```

Comment:
You could also add assertions for the other item in watched and one for verifying the watchlist contains what you expect.